hotsos/core/host_helpers/systemd.py

```diff
@@ -100,7 +100,7 @@ def start_time(self):
             if len(sections) == 0:
                 log.warning("no active status found for %s.service (state=%s)",
                             self.name, self.state)
-                return
+                return None
```

Comment:
`None` is the default value, right? Are we emphasising explicit returns? Maybe, a Coding Guidelines, if one doesn't exist, might be useful for future (perhaps Canonical-wide given the amount of Python we write).